src/main/java/org/sopt/watchapedia/domain/production/domain/Production.java

```diff
@@ -0,0 +1,33 @@
+package org.sopt.watchapedia.domain.production.domain;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@NoArgsConstructor
```

Comment:
엔티티 생성 시 빈 생성자 사용을 방지하기 위해 접근 제한을 protected로 설정하는 것이 좋을 것 같습니다.
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```